package/setup.py

```diff
@@ -650,6 +643,10 @@ def long_description(readme):
               'AMBER': [
                   'netCDF4>=1.0',  # for fast AMBER writing, also needs HDF5
               ],
+              'extra_formats': [   # additional file formats
+                  'netCDF4>=1.0',  # for fast AMBER writing, also needs HDF5
+                  'h5py>=2.10',    # H5MD
+                  ],
```

Comment:
possible add here
* gsd (unless we just install it anyway)
* chemfiles

Reply:
gsd is an install requires for now so it shouldn't also be going into extra requires

chemfiles might be an option if we want

Reply:
I added chemfiles>=0.10